src/Avalonia.Controls/NativeMenuItem.cs

```diff
@@ -169,4 +182,11 @@ void INativeMenuItemExporterEventsImplBridge.RaiseClicked()
             }
         }
     }
+    
+    public enum NativeMenuItemToggleType
+    {
+        None,
+        CheckBox,
+        Radio
```

Comment:
Not sure how we would acheive this on OSX exactly?

what defines a the group of exclusivity? seperators?

Can the logic be implemented in the `NativeMenu` class and then the exporters simply show if its checked or not?

Reply:
I think for now we can treat it as a special kind of checkbox and leave the uniqueness handing to the user code. For OSX we'll need to change `onStateImage` property so a radio dot.

Reply:
`[NSImage imageNamed:@"NSMenuRadio"]` should be appropriate